src/Microsoft.ML.LightGBM/LightGbmCatalog.cs

```diff
@@ -55,9 +55,9 @@ public static class LightGbmExtensions
         /// Predict a target using a decision tree binary classification model trained with the <see cref="LightGbmBinaryTrainer"/>.
         /// </summary>
         /// <param name="catalog">The <see cref="BinaryClassificationCatalog"/>.</param>
-        /// <param name="labelColumn">The labelColumn column.</param>
-        /// <param name="featureColumn">The features column.</param>
-        /// <param name="weights">The weights column.</param>
+        /// <param name="labelColumn">The name of the label column.</param>
+        /// <param name="featureColumn">The name of the features column.</param>
+        /// <param name="weights">The name of the weights column.</param>
```

Comment:
>the [](start = 46, length = 3)

optional weights column

Reply:
across whole file

---
In reply to: [255809893](https://github.com/dotnet/machinelearning/pull/2432#discussion_r255809893) [](ancestors = 255809893)